packages/site/src/providers/AtlantisPreviewEditorProvider.tsx

```diff
@@ -186,6 +192,9 @@ export const AtlantisPreviewEditorProvider = ({
 
                 ${transpiledCode}
 
+            function RootWrapper() {
```

Comment:
Awesome that you figured this out on your own. It's just code! I think this file can be cleaned up and improved, but overall I think we're onto something here! :)